libs/@local/hash-graph-client/typescript/openapitools.json

```diff
@@ -1,14 +1,14 @@
 {
-  "$schema": "../../../node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "$schema": "../../../../node_modules/@openapitools/openapi-generator-cli/config.schema.json",
   "spaces": 2,
   "generator-cli": {
     "version": "6.6.0",
     "generators": {
       "ts": {
         "generateAliasAsModel": true,
         "generatorName": "typescript-axios",
-        "output": "#{cwd}/../hash-graph-client",
-        "inputSpec": "../../../apps/hash-graph/openapi/openapi.json",
+        "output": "#{cwd}/../typescript",
```

Comment:
Isn't this the same as `"#{cwd}"`?

Reply:
I thought the same but was anxious to change the previously configured value (which also points to `#{cwd}`). I'll change it an see what happens.

Reply:
fixed in [4367f05](https://github.com/hashintel/hash/pull/2841/commits/4367f052c76d88b83b2be7386cc6db1d5abe57e2)